src/07-projects/project-items.apib

```diff
@@ -1,6 +1,6 @@
 ## Project items [/projectItems]
 
-Project items are added to a project and they affect the profitability. This information is only available for account administrators and users with full project admin access.
+Project items are added to a project and they affect the profitability. This information is only available for users with profit on projects access.
```

Comment:
PP/Nitpick:
IMHO `users with access to profit on projects` could perhaps be a bit clearer ?